File: Cpp/Day 8/inheritance/Assi1/date.cpp
#include"date.h"

    Date::Date()
    {
        d=m=1;
        y=2000;
    }
    Date::Date(int d,int m,int y)
    {
        this->d=d;
        this->m=m;
        this->y=y;
    }
    void Date::display()
    {
        cout<<endl<<"Date: "<<d<<"/"<<m<<"/"<<y;
    }

File: Java lab/Day 4 oops in java 3/Assi 4/Date.java
class Date
{
	private static int cnt;
	private int dd,mm,yy;
	public Date(){	cnt++; System.out.println(cnt+" th Object of date class is Created");}
	public Date(int dd,int mm,int yy)
	{	this.dd=dd;this.mm=mm;this.yy=yy; cnt++; System.out.println(cnt+" th Object of date class is Created");}
	public Date(Date d)
	{	this.dd=d.dd;this.mm=d.mm;this.yy=d.yy;	cnt++; System.out.println(cnt+" th Object of date class is Created");}
	public void ShowDate()
	{	System.out.println(dd+"/"+mm+"/"+yy);}
}